tests/operators/test_agnostic_load_file.py

```diff
@@ -469,6 +469,80 @@ def test_aql_gcs_file_to_postgres(self):
         )
         assert df.iloc[0].to_dict()["sell"] == 142.0
 
+    def test_aql_gcs_file_to_postgres_ndjson_with_string_path(self):
+        self.upload_blob()
+        OUTPUT_TABLE_NAME = "expected_table_from_gcs_csv"
+
+        self.hook_target = PostgresHook(
+            postgres_conn_id="postgres_conn", schema="pagila"
+        )
+
+        # Drop target table
+        drop_table_postgres(OUTPUT_TABLE_NAME, self.hook_target.get_conn())
+
+        self.create_and_run_task(
+            load_file,
+            (),
+            {
+                "path": "gs://dag-authoring/github/github_nested_000000000007.ndjson",
+                "file_conn_id": "",
+                "output_table": Table(
+                    OUTPUT_TABLE_NAME,
+                    database="pagila",
+                    conn_id="postgres_conn",
+                    schema="public",
+                ),
+                "flatten_ndjson": {
+                    "repository": "repository.url",
+                    "actor_attributes": "actor_attributes.company",
+                    "payload": "payload.size",
+                },
+            },
+        )
```

Comment:
Let's move away from `create_and_run_task`. It's constricting as we can't have multiple tasks that depend on eachother. Just make the dag and then use `test_utils.run_dag(self.dag)`

Reply:
Fixed.